packages/@react-spectrum/list/src/styles.css

```diff
@@ -599,7 +658,54 @@
 
 .react-spectrum-ListView {
   &:focus-ring {
-    border-color: var(--spectrum-global-color-blue-500);
-    box-shadow: inset 0 0 0 1px var(--spectrum-table-cell-border-color-key-focus);
+    border-color: var(--spectrum-listview-border-color-key-focus);
+    box-shadow: inset 0 0 0 1px var(--spectrum-listview-border-color-key-focus);
   }
 }
+
+@media (forced-colors: active) {
+  .react-spectrum-ListView,
+  .react-spectrum-ListViewItem {
```

Comment:
why do we need both selectors? we should be able to just use the top level one
`.react-spectrum-ListView`

Reply:
Drag and drop previews for the `.react-spectrum-ListViewItem` are no longer descendants of `.react-spectrum-ListView`, and they end up being rendered without backgrounds and borders with `forced-color-adjust: none`

Reply:
ahhhh the drop previews are standalone items, ok, that makes sense
thanks for explaining